home-manager-module.nix

```diff
@@ -0,0 +1,100 @@
+{ systemdLib }:
+{
+  config,
+  osConfig,
+  options,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.virtualisation.user.quadlet;
+  quadletUtils = import ./utils.nix {
+    inherit lib;
+    systemdLib = systemdLib { inherit lib config pkgs; };
+  };
+  containerOpts = lib.types.submodule (import ./container.nix { inherit quadletUtils; });
+  networkOpts = lib.types.submodule (import ./network.nix { inherit quadletUtils pkgs; });
+in
+{
+  options.virtualisation.user.quadlet = {
+    autoUpdate = {
+      enable = lib.mkOption {
+        type = lib.types.bool;
+        default = false;
+      };
+      calendar = lib.mkOption {
```

Comment:
Nice. Do you want to add the same timer for rootful quadlet?

Reply:
Sure can do.